node/os_test.ts

```diff
@@ -173,17 +173,6 @@ Deno.test({
   },
 });
 
-Deno.test({
```

Comment:
This was never right, IDK why this was done this way

Reply:
Right. These should be numbers, instead of strings